javascript/javascript1/week2/event.js

```diff
@@ -0,0 +1,30 @@
+const daysArray = new Array('Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'); // this code is applicable for Day
+const todaysDate = new Date()
+const day1 = todaysDate.getDay(); //** this line is important
+
+//console.log(day1); // expected output: 4
+
+function remainder(addedTotalDays) {
+    return addedTotalDays % 7;
+}
```

Comment:
It is very simple and you used it one time, so you don't have to make a function for it